packages/lesswrong/components/posts/usePostsItem.tsx

```diff
@@ -76,6 +76,8 @@ export type PostsItemConfig = {
   forceSticky?: boolean,
   showReadCheckbox?: boolean,
   showKarma?: boolean,
+  /** If the post is curated, whether to show the curation date instead of the postedAt date. Default true. */
+  useCuratedDate?: boolean,
```

Comment:
We'd prefer for this to be default `false` on EAF, since showing the curated date instead of the posted date can be confusing for users. Based on some conversation I saw in slack, my guess is that the LW default `true` case is more of the special case.

Reply:
Seems fine to change it, but this PR isn't intended to be a change in behavior (except in the specific context of PostsTimeBlock).